compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -695,7 +707,12 @@ static void PrintStub(
       case BLOCKING_CALL:
         GRPC_CODEGEN_CHECK(!client_streaming)
             << "Blocking client interface with client streaming is unavailable";
-        if (server_streaming) {
+        if (client_streaming && server_streaming) {
+          p->Print(
+              *vars,
+              "$BlockingClientCall$<$input_type$, $output_type$>\n"
```

Comment:
Let's go ahead and mark these `@ExperimentalApi` for at least one release? The code here is pretty tight. The main thing I could see that could change is `BlockingClientCall<?, RespT>` becoming `BlockingClientCall<ReqT, RespT>`, because that seems the only _choice_ we have.

Although maybe we could also remove the `throws InterruptedException` from server streaming, since we "know" it won't block. Dunno if that is worth it.

Reply:
The generated method is not `ExperimentalApi`.

Reply:
Added experimental annotation to generated code.